ws-nextjs-app/package.json

```diff
@@ -13,15 +13,21 @@
     "start": "next start -p 7081 && NODE_ENV=production",
     "test": "jest --ci --colors"
   },
+  "resolutions": {
+    "@types/react": "18.0.33"
+  },
```

Comment:
The latest version that is bundled with Next 13 seems to have compilation errors. Forcing this to be the same version we have in the Express app.